tools/check-rsyslog-version

```diff
@@ -15,14 +15,16 @@ else
   MINIMAL_RSYSLOG_VERSION="5.7.1"
 fi
 
+CANONIFIED_VERSION=$(echo ${MINIMAL_RSYSLOG_VERSION} | sed -r 's/\./_/g')
+
```

Comment:
Yes, the sed command is similar to the examples for AIX 5.3 (http://www.ibm.com/support/knowledgecenter/ssw_aix_53/com.ibm.aix.cmds/doc/aixcmds5/sed.htm), and the $() works in ksh (https://www.ibm.com/developerworks/aix/library/au-kornshellscripting/)